app/js/components/answers/Show.react.js

```diff
@@ -48,30 +54,45 @@ class AllAnswers extends React.Component {
 
    render () {
      let answer = this.props.answer || {};
+     let question = this.props.question || {};
```

Comment:
Declaring so many variables would be hard to test. Is there a way we can reduce the number of variables being declared or have it in form of an Object?